core/src/main/java/de/danoeh/antennapod/core/storage/PodDBAdapter.java

```diff
@@ -209,7 +214,7 @@
             + KEY_PLAYBACK_COMPLETION_DATE + " INTEGER,"
             + KEY_FEEDITEM + " INTEGER,"
             + KEY_PLAYED_DURATION + " INTEGER,"
-            + KEY_AUTO_DOWNLOAD + " INTEGER)";
```

Comment:
I'm not sure why, but auto download is defined in FeedItem and FeedMedia, but only used by FeedItem
Basically, the column is never read